server/player.lua

```diff
@@ -539,14 +539,17 @@ function QBCore.Player.DeleteCharacter(source, citizenid)
     local result = MySQL.scalar.await('SELECT license FROM players where citizenid = ?', { citizenid })
     if license == result then
         local query = "DELETE FROM %s WHERE citizenid = ?"
-        local tableCount = #playertables
+        local tableCount = #playertables + 1 -- Add 1 for the apartments table
```

Comment:
There's no reason for this. Your PR only requires 1 line addition.. add `apartments` to the playertables (https://github.com/qbcore-framework/qb-core/pull/962/files#diff-7bac2e2b00ff0a0b87f47152e9794e29e49043b171068b6d93fd65572dabac77R522) as the rest of it requires no code changes (the `citizenid` column is the same as does the query format).